Assets/Amplitude/Amplitude.cs

```diff
@@ -838,6 +838,23 @@ public class Amplitude {
 #endif
 	}
 
+	/// <summary>
+	/// iOS: 
+	/// This method has no equivalent in iOS. See comments in `useAdvertisingIdForDeviceId()`.
+	/// 
+	/// Android: If this method is called, the setting for Advertising ID will be used first for the user's device ID,
```

Comment:
If useAppSetIdForDeviceId is called, Advertising ID will be used first for device ID, instead of App set id?

Reply:
yeah, if both are set, ad id will be used as a first choice.

Reply:
ok I got it now. Thanks.